pkg/integrations/postgres_exporter/postgres_exporter.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	DisableSettingsMetrics bool     `yaml:"disable_settings_metrics,omitempty"`
 	AutodiscoverDatabases  bool     `yaml:"autodiscover_databases,omitempty"`
 	ExcludeDatabases       []string `yaml:"exclude_databases,omitempty"`
+	IncludeDatabases       []string `yaml:"include_databases,omitempty"`
```

Comment:
I deliberately did *not* add the "metricPrefix" option which is new in the latest version of the exporter. My thinking is that we want to be opinionated about metric names/namespaces in the agent.